doc/install.rst

```diff
@@ -221,12 +221,11 @@ command::
     $ sudo port install py36-scikit-learn
 
 
-Canopy and Anaconda for all supported platforms
------------------------------------------------
+Anaconda and Enthought Deployment Manager for all supported platforms
+---------------------------------------------------------------------
 
-`Canopy
-<https://www.enthought.com/products/canopy>`_ and `Anaconda
-<https://www.anaconda.com/download>`_ both ship a recent
+`Anaconda <https://www.anaconda.com/download>`_ and
+`Enthought Deployment Manager <https://assets.enthought.com/downloads/>`_ both ship a recent
```

Comment:
Since the enthought is not "recent"

> both ship with scikit-learn in addition

And wrap this so there is <80 characters per line.

Reply:
@thomasjpfan Thanks for the comments!